jib-maven-plugin-extension-api/build.gradle

```diff
@@ -0,0 +1,40 @@
+plugins {
+  id 'net.researchgate.release'
+  id 'maven-publish'
+}
+
+dependencies {
+  api project(':jib-build-plan')
```

Comment:
So I think this part is weird... We don't actually want a direct dependency on the source like this do we?

In the end we should depend on the publicly available `jib-build-plan` obviously we can't do that now since we never released anything.

Reply:
If we assume we published `com.google.cloud.tools:jib-build-plan:0.1.0`, I think `api 'com.google.cloud.tools:jib-build-plan:0.1.0'` is the correct way to go, right? Because the extension dev should be able to access and use those classes in `jib-build-plan`, isn't it? (I assume the dev will only declare this dependency of this Maven extension API, after which I expect the dev can make use of all the classes in both `jib-build-plan` and `jib-maven-plugin-extension-api` plus the maven-core API.) Am I getting this right?

So, given that we haven't released `jib-build-plan`, what do you think is best to move forward at the moment, maybe a temporary measure for the sake of moving forward?